docs/compiling-mono/linux/index.md

```diff
@@ -76,7 +78,7 @@ sudo mkdir $PREFIX
 sudo chown -R `whoami` $PREFIX
 
 # Ensure that all required packages are installed.
-sudo apt-get install git autoconf libtool automake build-essential mono-devel gettext
+sudo apt-get install git autoconf libtool libtool-bin automake build-essential mono-devel gettext
```

Comment:
As noted in https://github.com/mono/website/pull/124#issuecomment-111735382, this won't work on earlier Ubuntu releases. Can you please remove it from here?